changelog.d/10820.misc

```diff
@@ -0,0 +1 @@
+Made a change so that posting an event with "\u0000" no longer causes an internal server error.
```

Comment:
'Made a change' sounds a bit soft (you fixed a bug — don't understate it :)) and I would probably say a 'null byte' rather than `"\u0000"`. I am probably not the best wordsmith around, however :innocent:.

```suggestion
Fix a bug where sending an `m.room.text` event containing a null byte would cause an internal server error.
```